### ISSUE TITLE
cf-jwt folder missing from bundle

### ISSUE THREAD

**Issue by djmarland** [block65/webcrypto-web-push#1]
Hello,

In trying to deploy my app to cloudflare I am receiving this error:

```
23:41:53.259 | [vite-plugin-pwa:build] Could not resolve "./cf-jwt/main.js" from "node_modules/@block65/webcrypto-web-push/dist/lib/vapid.js"
-- | --
23:41:53.259 | file: /opt/buildhome/repo/node_modules/@block65/webcrypto-web-push/dist/lib/vapid.js
23:41:53.262 | error during build:
23:41:53.263 | RollupError: Could not resolve "./cf-jwt/main.js" from "node_modules/@block65/webcrypto-web-push/dist/lib/vapid.js"
23:41:53.263 | at error (file:///opt/buildhome/repo/node_modules/rollup/dist/es/shared/node-entry.js:2245:30)
```

It seems that the npm package does not include the file at `dist/lib/cf-jwt`
https://www.npmjs.com/package/@block65/webcrypto-web-push?activeTab=code

But I do see the `cf-jwt` folder in github https://github.com/block65/webcrypto-web-push/tree/master/lib/cf-jwt

Should it be included in the released bundle?

Thanks

**Comment by maxholman** [comment]
Thanks for the report! You are right, that directory is not included as it's not covered by the `files` glob in package.json.  I'll fix and get a new release out.

**Comment by maxholman** [comment]
Published as v0.3.1

**Comment by djmarland** [comment]
Thank you. You are amazing.
